proxy/envoy/testdata/envoy-v0-auth.json.golden

```diff
@@ -0,0 +1,416 @@
+                }
+              ]
+            }
+          }
+        }
+      ],
+      "bind_to_port": false
+    },
+    {
+      "address": "tcp://10.1.1.0:1081",
+      "filters": [
+        {
+          "type": "read",
+          "name": "http_connection_manager",
+          "config": {
+            "codec_type": "auto",
+            "stat_prefix": "http",
+            "route_config": {
+              "virtual_hosts": [
+                {
+                  "name": "hello.default.svc.cluster.local|http-status",
+                  "domains": [
+                    "hello:81",
+                    "hello.default:81",
+                    "hello.default.svc:81",
+                    "hello.default.svc.cluster:81",
+                    "hello.default.svc.cluster.local:81",
+                    "10.1.0.0:81",
+                    "10.1.1.0:1081"
+                  ],
+                  "routes": [
+                    {
+                      "prefix": "/",
+                      "cluster": "in.1081",
+                      "opaque_config": {
+                        "mixer_control": "on",
+                        "mixer_forward": "off"
+                      }
+                    }
+                  ]
+                }
+              ]
+            },
+            "filters": [
+              {
+                "type": "decoder",
+                "name": "mixer",
+                "config": {
+                  "mixer_server": "mixer:9091",
+                  "mixer_attributes": {
+                    "target.service": "hello.default.svc.cluster.local",
+                    "target.uid": "10.1.1.0"
+                  },
+                  "forward_attributes": {
+                    "source.service": "hello.default.svc.cluster.local",
+                    "source.uid": "10.1.1.0"
+                  }
+                }
+              },
+              {
+                "type": "decoder",
+                "name": "router",
+                "config": {}
+              }
+            ],
+            "access_log": [
+              {
+                "path": "/dev/stdout"
+              }
+            ]
+          }
+        }
+      ],
+      "ssl_context": {
+        "cert_chain_file": "/etc/certs/cert-chain.pem",
+        "private_key_file": "/etc/certs/key.pem",
+        "ca_cert_file": "/etc/certs/root-cert.pem"
+      },
+      "bind_to_port": false
+    },
+    {
+      "address": "tcp://10.1.1.0:1090",
+      "filters": [
+        {
+          "type": "read",
+          "name": "tcp_proxy",
+          "config": {
+            "stat_prefix": "tcp",
+            "route_config": {
+              "routes": [
+                {
+                  "cluster": "in.1090",
+                  "destination_ip_list": [
+                    "10.1.1.0/32"
+                  ]
+                }
+              ]
+            }
+          }
+        }
+      ],
+      "bind_to_port": false
+    },
+    {
+      "address": "tcp://10.1.1.0:80",
```

Comment:
@kyessenov (the question is not related to this PR) why do we use 10.1.1.0 which is an invalid IP if it is not a wildcard? Maybe using something like 10.1.1.2?